images/api-resource-collector/Dockerfile.ci

```diff
@@ -0,0 +1,18 @@
+# Step one: build fetcher
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
```

Comment:
You can already use `FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder`, other components use 1.13 in the meantime. Doesn't need to be as part of this patch, but it would be nice to fix soon, I've already seen issues with some deps that use the new errors API..

Reply:
Actually I'm removing this image when I update CO to use sub-commands